tests/auth_tests/test_forms.py

```diff
@@ -261,6 +261,21 @@ class Meta(UserCreationForm.Meta):
             ['The password is too similar to the first name.'],
         )
 
+    def test_autocomplete(self):
+        form = UserCreationForm()
+
+        tests = (
+            ('username', 'username'),
+            ('password1', 'new-password'),
+            ('password2', 'new-password')
+        )
+        for field_name, autocomplete in tests:
+            with self.subTest(field_name=field_name, autocomplete=autocomplete):
+                self.assertEqual(
+                    form.fields[field_name].widget.attrs.get('autocomplete', ''),
+                    autocomplete
```

Comment:
Don't forget the trailing commas!

However, I also expect that this can fit on one line - we allow 119 chars, and it is not too complicated to understand.